stories/Truncate.stories.js

```diff
@@ -40,6 +40,7 @@ stories.add('default', () => (
       Truncate by Splitter:
       <br />
       <Truncate splitter="@">longemailaddress@gmail.com</Truncate>
+      <Truncate splitter="@">a@myemail.com</Truncate>
```

Comment:
Can we add some additional examples here so we can visually see how this looks under more scenarios?

Reply:
Copying the examples from the Editable Fields story.